### ISSUE TITLE
train the model for other languages

### ISSUE THREAD

**Issue by zarakolagar120** [google-research/pegasus#169]
Hi, great job! I was wondering if it is possible to train this model on other languages? if yes, could you please let me know what might be the requirements to be able to use it for other languages?

**Comment by JingqingZ** [comment]
Hi, just quick thoughts

1. Collect a sufficiently large text corpus for the language.
2. Modify tokenization accordingly.
3. Prepare GPUs/TPUs and start the training.

**Comment by OpUs-Nebula** [comment]
Hello @JingqingZ, in what way does tokenization need to be modified? Isn't sentencepiece language independent?

**Comment by JingqingZ** [comment]
Good point, yes, sentencepiece is language independent. So using sentencepiece should be fine.

**Comment by Haffi112** [comment]
Has anyone gotten the model to work for another language and documented the process?

**Comment by Darshan2104** [comment]
I have one thought in my mind what if we directly fine-tune this pre-trained model in another language other than English! will it work or should I change word embedding according to my input language? any thoughts are welcomed.

**Comment by Steereo** [comment]
Hi, just as an input, there is a large corpus with about 44 languages to be used called XL-sum. 
If I do end up^progressing in this task in anyway I'll be sure to link it
